corgi/tasks/errata_tool.py

```diff
@@ -36,12 +36,12 @@ def save_variant_cdn_repo_mapping() -> None:
 
 @app.task(base=Singleton, autoretry_for=RETRYABLE_ERRORS, retry_kwargs=RETRY_KWARGS)
 def slow_save_errata_product_taxonomy(erratum_id: int):
-    logger.info(f"slow_save_errata_product_taxonomy called for {erratum_id}")
-    relation_build_ids = _get_relation_build_ids(erratum_id)
-    for b in relation_build_ids:
-        logger.info("Saving product taxonomy for build %s", b)
+    logger.info(f"save_errata_product_taxonomy called for {erratum_id}")
```

Comment:
Minor nit: looks like a bad merge conflict resolution here.
```suggestion
    logger.info(f"slow_save_errata_product_taxonomy called for {erratum_id}")
```

Reply:
updated.